tasks/kafka-cfg.yml

```diff
@@ -50,7 +50,7 @@
     - kafka-install
 
 - name: "Link alternate logs directory and touch output files"
-  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s /var/log/kafka /usr/local/kafka/logs && touch /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out && chown kafka:kafka /var/log/kafka/state-change.log /var/log/kafka/kafkaServer.out"
+  shell: "( test ! -e /usr/local/kafka/logs || rm -rf /usr/local/kafka/logs ) && ln -s {{ kafka_log_dir }} /usr/local/kafka/logs && touch {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out && chown {{ kafka_user }}:{{ kafka_group }} {{ kafka_log_dir }}/state-change.log {{ kafka_log_dir }}/kafkaServer.out"
```

Comment:
but I can still see @fissurezone 's choice for that role .. seeing other hits: https://www.google.ch/search?q=ansible%20kafka&oq=ansible%20kafka
maybe the hp one could be good, just looks quite outdated .. the 3.hit (lloydmeta) is only for centOS ...